### PR TITLE
 Introduced panels components for Automations, Languages, Plugins, Roles and Users - Fixed #53

### LINKED ISSUE #53: InfoBox component requires a "title" attribute for i18n
Under the icon it shows `action` property as a title so if you translate it by i18n `openWindow()` doesn't work anymore.

### PR DESCRIPTION
**Pull request by mcicolella**

### CODE REVIEW

File: src/components/InfoBox.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default {
   data () {
     return {}
   },
-  props: ['color', 'action', 'icon'],
+  props: ['color', 'action', 'icon', 'title'],
```
Comment (P3trur0) [inline src/components/InfoBox.vue]: What do you think to transform this props in other properties of the Vuex global state?

File: src/components/LanguagesPanel.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<template>
+    <div v-if="languagesOpen" class="current-section">
+        <md-toolbar class="md-accent">
+          <md-content class="small-icon" @click="openSettings">
+            <md-icon>undo</md-icon>
+          </md-content>
+          <md-content class="small-icon" @click="closeWindow">
+            <md-icon>clear</md-icon>
+          </md-content> 
+          <h3 class="md-title">Languages</h3>
```
Comment (P3trur0) [inline src/components/LanguagesPanel.vue]: Maybe this is a label to use with I18n locales

File: src/components/AutomationsPanel.vue [hunk 3]
```diff
@@ -0,0 +1,67 @@
+<template>
+    <div v-if="automationsOpen" class="current-section">
+        <md-toolbar class="md-accent">
+          <md-content class="small-icon" @click="openSettings">
+            <md-icon>undo</md-icon>
+          </md-content>
+          <md-content class="small-icon" @click="closeWindow">
+            <md-icon>clear</md-icon>
+          </md-content> 
+          <h3 class="md-title">Automations</h3>
```
Comment (P3trur0) [inline src/components/AutomationsPanel.vue]: Maybe this is a label to use with I18n locales

File: src/components/PluginsPanel.vue [hunk 4]
```diff
@@ -0,0 +1,67 @@
+<template>
+    <div v-if="pluginsOpen" class="current-section">
+        <md-toolbar class="md-accent">
+          <md-content class="small-icon" @click="openSettings">
+            <md-icon>undo</md-icon>
+          </md-content>
+          <md-content class="small-icon" @click="closeWindow">
+            <md-icon>clear</md-icon>
+          </md-content> 
+          <h3 class="md-title">Plugins</h3>
```
Comment (P3trur0) [inline src/components/PluginsPanel.vue]: Maybe this is a label to use with I18n locales

File: src/components/RolesPanel.vue [hunk 5]
```diff
@@ -0,0 +1,67 @@
+<template>
+    <div v-if="rolesOpen" class="current-section">
+        <md-toolbar class="md-accent">
+          <md-content class="small-icon" @click="openSettings">
+            <md-icon>undo</md-icon>
+          </md-content>
+          <md-content class="small-icon" @click="closeWindow">
+            <md-icon>clear</md-icon>
+          </md-content> 
+          <h3 class="md-title">Roles</h3>
```
Comment (P3trur0) [inline src/components/RolesPanel.vue]: Maybe this is a label to use with I18n locales

File: src/components/UsersPanel.vue [hunk 6]
```diff
@@ -0,0 +1,67 @@
+<template>
+    <div v-if="usersOpen" class="current-section">
+        <md-toolbar class="md-accent">
+          <md-content class="small-icon" @click="openSettings">
+            <md-icon>undo</md-icon>
+          </md-content>
+          <md-content class="small-icon" @click="closeWindow">
+            <md-icon>clear</md-icon>
+          </md-content> 
+          <h3 class="md-title">Users</h3>
```
Comment (P3trur0) [inline src/components/UsersPanel.vue]: Maybe this is a label to use with I18n locales

**Comment by mcicolella** [comment]
@P3trur0 Thanks, I'll open specific issues for your suggestions.

**Comment by P3trur0** [comment]
@mcicolella isn't that enough to stay with this PR?

**Comment by mcicolella** [comment]
@P3trur0 for me 'props' to 'Vuex' global state transformation should be managed in another PR to include all components that require this change. Ok for i18n labels in this PR.

**Comment by P3trur0** [comment]
@mcicolella I agree

**Pull request merged**